src/main/java/io/strimzi/kafka/quotas/StaticQuotaCallback.java

```diff
@@ -133,11 +133,17 @@ public void configure(Map<String, ?> configs) {
         long storageCheckInterval = config.getStorageCheckInterval();
         if (storageCheckInterval > 0L) {
             final Optional<Long> availableBytesLimitConfig = config.getAvailableBytesLimit();
-
+            final Optional<Double> availableRatioLimit = config.getAvailableRatioLimit();
+            if (availableBytesLimitConfig.isPresent() && availableRatioLimit.isPresent()) {
+                throw new IllegalStateException("both limit types configured, only configure one");
```

Comment:
1. Capital letter.
2. Give the names of the config parameters so it's completely clear what's wrong.